src/braket/device_schema/device_connectivity.py

```diff
@@ -38,4 +38,4 @@ class DeviceConnectivity(BaseModel):
     """
 
     fullyConnected: bool
-    connectivityGraph: dict
+    connectivityGraph: dict[str, list[str]]
```

Comment:
Is there any backward-compatibility concern here? Is there a specific problem this is fixing, or is it just hygiene?

Reply:
Discussed offline, this is the expected format for Braket. This will be compatible with existing schemas but just more strictly defines this based around the current expected format